core/scss/_props.scss

```diff
@@ -32,14 +25,10 @@
   /** Theme colors */
   @each $name, $color in map-merge($theme-colors, $social-colors) {
     --#{$prefix}#{$name}: #{$color};
-    --#{$prefix}#{$name}-rgb: #{to-rgb($color)};
     --#{$prefix}#{$name}-fg: #{if(contrast-ratio($color) > $min-contrast-ratio, var(--#{$prefix}light), var(--#{$prefix}dark))};
-    --#{$prefix}#{$name}-darken: #{theme-color-darker($color)};
-    --#{$prefix}#{$name}-darken: color-mix(in oklab, var(--#{$prefix}#{$name}), transparent 20%);
-    --#{$prefix}#{$name}-lt: #{theme-color-lighter($color)};
-    --#{$prefix}#{$name}-lt: color-mix(in oklab, var(--#{$prefix}#{$name}) 10%, transparent);
-    --#{$prefix}#{$name}-200: color-mix(in oklab, var(--#{$prefix}#{$name}) 20%, transparent);
-    --#{$prefix}#{$name}-lt-rgb: #{to-rgb(theme-color-lighter($color))};
```

Comment:
Removing the `-lt` classes is a BC break. I guess they were used quite often, as they were referenced in the docs for a long time. Not sure about the other ones, but in a strict semantic world you should add the new classes and only in the next major release remove the old classes, so all users have the time to update their code bases.

Real world example: the `-lt` classes are used in multiple plugins for Kimai, which are made by different developers and the releases cannot be synced. So if I update Tabler, frontend stuff will 100% surely break in several places.

Please re-add/keep the `-lt` classes as an alias for the new`-lightest` classes.